### ISSUE TITLE
Broken HelpContentURI in test assets

### ISSUE THREAD

**Issue by xtqqczze** [PowerShell/PowerShell#8690]
In `test\powershell\engine\Help\assets\Microsoft.PowerShell.LocalAccounts_8e362604-2c0b-448f-a414-a6a690a644e2_HelpInfo.xml`

This URI is broken:
http://go.microsoft.com/fwlink/?LinkId=717973

302 (Moved Temporarily) to http://download.microsoft.com/download/2/3/0/23073AB1-2A89-45F0-8673-FBBA4636AF06/
where server responds 403 (Forbidden)

**Comment by iSazonov** [comment]
@SteveL-MSFT Could you please assign right man?

**Comment by SteveL-MSFT** [comment]
cc @adityapatwardhan

**Comment by adityapatwardhan** [comment]
@xtqqczze The URI is not broken. The behavior is expected as it redirects to a container.

```PowerShell
PS> Update-Help -Module Microsoft.PowerShell.LocalAccounts -Verbose -Force
VERBOSE: Resolving URI: "http://go.microsoft.com/fwlink/?LinkId=717973"
VERBOSE: Your connection has been redirected to the following URI:
"http://download.microsoft.com/download/2/3/0/23073AB1-2A89-45F0-8673-FBBA4636AF06/"
VERBOSE: Performing the operation "Update-Help" on target "Microsoft.PowerShell.LocalAccounts, Current Version:
5.1.1.0, Available Version: 5.1.1.0, UICulture: en-US".
VERBOSE: Microsoft.PowerShell.LocalAccounts: Updated
C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules\Microsoft.PowerShell.LocalAccounts\1.0.0.0\en-US\Microsoft.Powershel
l.LocalAccounts.dll-Help.xml. Culture en-US Version 5.1.1.0
```

**Comment by xtqqczze** [comment]
@adityapatwardhan @iSazonov Works for me too.
```
PS C:\WINDOWS\system32> $PSVersionTable.PSEdition
Desktop
```
```
PS C:\WINDOWS\system32> Update-Help -Module Microsoft.PowerShell.LocalAccounts -UICulture en-US -Verbose -Force
VERBOSE: Resolving URI: "http://go.microsoft.com/fwlink/?LinkId=717973"
VERBOSE: Your connection has been redirected to the following URI:
"https://download.microsoft.com/download/2/3/0/23073AB1-2A89-45F0-8673-FBBA4636AF06/"
VERBOSE: Performing the operation "Update-Help" on target "Microsoft.PowerShell.LocalAccounts, Current Version:
5.1.1.0, Available Version: 5.1.1.0, UICulture: en-US".
VERBOSE: Microsoft.PowerShell.LocalAccounts: Updated
C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules\Microsoft.PowerShell.LocalAccounts\1.0.0.0\en-US\Microsoft.Powershel
l.LocalAccounts.dll-Help.xml. Culture en-US Version 5.1.1.0
```
